MediaBrowser.Providers/Plugins/MusicBrainz/MusicBrainzAlbumProvider.cs

```diff
@@ -341,7 +341,7 @@ public static IEnumerable<ReleaseResult> Parse(XmlReader reader)
 
                                     using (var subReader = reader.ReadSubtree())
                                     {
-                                        return ParseReleaseList(subReader).ToList();
+                                        return ParseReleaseList(subReader);
```

Comment:
You're deferring execution here and `subReader` may or may not be disposed when the enumeration happens.

Reply:
You're 100% right and I'll fix.